task2.py

```diff
@@ -0,0 +1,22 @@
+def check_triangle(a, b, c):
+    if a + b <= c or a + c <= b or b + c <= a:
+        return False
+
+    if a == b == c:
+        return "Равносторонний треугольник"
+    elif a == b or a == c or b == c:
+        return "Равнобедренный треугольник"
+    else:
+        return "Разносторонний треугольник"
```

Comment:
Особенность функции заключается в том, что `return` завершает работу программы. Значит, не обязательно использовать конструкции `else`, так как код все равно не дойдет до тех строк, если предыдущее условие оказалось истинным.
```suggestion
    if a == b or a == c or b == c:
        return "Равнобедренный треугольник"

    return "Разносторонний треугольник"
```